packages/ui-components/src/assets/styles/kv-theme.default.scss

```diff
@@ -108,8 +107,7 @@ $colors: (
 		'step-8': $neutral-color-0
 	),
 	'text': (
-		/* Steps move towards their contrasting color. */
-		'step-0': $neutral-color-0,
+		/* Steps move towards their contrasting color. */ 'step-0': $neutral-color-0,
```

Comment:
fix this

Reply:
Sorry about that, my autosave is formating that :pensive: